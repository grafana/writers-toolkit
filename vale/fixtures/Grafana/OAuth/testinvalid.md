Don't use:

- Oauth
- Oauth 2
- OAuth
- OAuth 2
