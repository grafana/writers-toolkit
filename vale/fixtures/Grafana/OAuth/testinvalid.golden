testinvalid.md:3:3:Grafana.OAuth:Use 'OAuth 2.0' instead of 'Oauth'.
testinvalid.md:4:3:Grafana.OAuth:Use 'OAuth 2.0' instead of 'Oauth 2'.
testinvalid.md:5:3:Grafana.OAuth:Use 'OAuth 2.0' instead of 'OAuth'.
testinvalid.md:6:3:Grafana.OAuth:Use 'OAuth 2.0' instead of 'OAuth 2'.
